packages/vscode-extension/src/debug/constants.ts

```diff
@@ -93,8 +95,18 @@ export const teamsAppIdPlaceholder = "${teamsAppId}";
 export const teamsAppInternalIdPlaceholder = "${teamsAppInternalId}";
 export const accountHintPlaceholder = "${account-hint}";
 
-export const openOutputPanelCommand = "command:fx-extension.showOutputChannel";
-export const openTerminalCommand = "command:workbench.action.terminal.focus";
+export const openOutputMessage = "Please check the output to see the details.";
```

Comment:
What about reuse the existing "teamstoolkit.localDebug.showDetail" string, like 
`export const openOutputMessage = util.format(
    localize("teamstoolkit.localDebug.showDetail"),
    localize("teamstoolkit.localDebug.outputPanel")
  )`

Reply:
Good idea, changed the output messages.